packages/server-core/src/user/login-token/login-token.hooks.ts

```diff
@@ -40,6 +44,14 @@ import {
   loginTokenResolver
 } from './login-token.resolvers'
 
+const checkIdentityProvider = async (context: HookContext): Promise<HookContext> => {
```

Comment:
The `HookContext` here should be imported from declarations.ts and also it should be like `HookContext<LoginTokenService>` for the correct typings to work.

You can have a look of how `restrictUserPatch` in `packages/server-core/src/user/user/user.hooks.ts` does it.

Reply:
Fixed